docs/concepts/vega-chain/data-nodes.md

```diff
@@ -0,0 +1,160 @@
+
+If you are running your own data node, you can choose to enable any/all of the protocols, to tailor to your needs. Data nodes run by validators are expected to provide GraphQL, gRPC, and REST, and reliably serve data.
+
+## Database
+The data node relies on PostgreSQL and IPFS. Postgres, an open source, relational database that supports relational and non-relational queries, is used to store and provide the data.
+
+IPFS is used to share decentralised history segments across a data node's network.
+
+:::note Go deeper
+* [About PostgreSQL ↗](https://www.postgresql.org/about/): Read about PostgreSQL and explore the documentation.
+* [About IPFS ↗](https://docs.ipfs.tech/concepts/what-is-ipfs/): Get familiar with IPFS.
+:::
+
+## Connect directly to database
+It's possible to connect directly to the Postgres database of a data node without running a data node yourself, if the data node you're using allows it. You'll need the connection details from the data node operator, such as the relevant address, username and password in order to connect directly to that node's database.
+
+Alternatively, you can run your own data node and use [decentralised history](#decentralised-history) to pull existing data into your new database, or start fresh.
+
+:::caution Reliability
+PostgreSQL database schemas are liable to change without warning and without deprecation.
+:::
+
+<!-- Link to the data node postgres information.-->
+
+## Data retention
+A data node's data retention time allows data node operator to store historical data for specific amounts of time.
+
+Data types are grouped into categories, and the retention time for each type can be changed in the data node's `config.toml` file. You can configure a data node so that all data older than any time period (e.g. `1m`, `1h`, `1h:22m:32s`, `1 month`) is deleted, or extend the default times for the retention mode you choose.
+
+Some data can be saved in detail, and in a less-detailed sampling. Take balances, for example: In addition to saving every balance change, the `conflated balances` keep a sample of a balance from once every hour. This allows the data node to have data for a long period of time, but taking up less space than saving every balance change over a year.
+
+The data node code's configuration includes a set of default retention time frames. Those time frames have been defined with an individual trader that wants to run their own data node, for their own use, in mind. A public data node that's intended to support querying archival information will need much longer data retention time frames.
+
+Whenever a data node starts up, the existing retention policy for each data type is output in the logs.
+
+### Data retention modes
+When starting a data node, you can choose one of three retention modes, depending on what you want the data node to do. 
```

Comment:
This should be:

When initializing a data node.

When starting a data node, the chosen retention model will be applied. After initialization, it's possible to change the retention mode, for example from standard to archive, or lite by changing the retention period setting in the configuration file.